client/state/sites/plans/test/selectors.js

```diff
@@ -695,4 +700,90 @@ describe( 'selectors', () => {
 			).toEqual( PLAN_PREMIUM );
 		} );
 	} );
+
+	describe( 'isSiteOnECommerceTrial()', () => {
```

Comment:
Minor, looks like we tend to use a `#` before each test description for the other tests here.